src/modules/Dashboard/components/ParticipantSnippet/ParticipantSnippet.types.ts

```diff
@@ -0,0 +1,10 @@
+import { BoxProps } from '@mui/material/Box/Box';
+import React from 'react';
+
+export type ParticipantSnippetProps<T extends React.ElementType> =
```

Comment:
In my opinion, it's better to import elements from 'react' individually rather than importing the entire library, like this: ``import { ElementType } from 'react';`` Please verify this approach in all other places.